dask/dataframe/core.py

```diff
@@ -2311,6 +2311,11 @@ def shift(self, periods=1, freq=None):
             freq = meta.freq
         return maybe_shift_divisions(out, periods, freq=freq)
 
+    @derived_from(pd.Index)
+    def to_series(self):
+        return self.map_partitions(M.to_series,
+                                   meta=self._meta.to_series())
```

Comment:
My guess is that if you leave off the `meta=` keyword here that it will probably do the right thing anyway.

Reply:
Yep, I tried that also and it worked.